backend/app/domain/thread.go

```diff
@@ -0,0 +1,76 @@
+package domain
+
+import (
+	"time"
+
+	"github.com/jumpei00/board/backend/app/params"
+)
+
+type Thread struct {
+	threadKey   string
+	title       string
+	contributer string
+	postDate    time.Time
+	views       int
+	sumComment  int
+}
+
+func NewThread(param *params.ThreadCreateDomainLayerParam) *Thread {
+	thread := &Thread{
+		title:       param.Title,
+		contributer: param.Contributer,
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`contributer` is a misspelling of `contributor` (misspell)